fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -84,20 +95,37 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
 
 	const [page, setPage] = useState(1);
 
-	/*const debouncedRunSearch = debounce(() => runSearch(page), 750); TODO need to check if needed for chef-search? if yes then how to improve implementing it*/
-
 	useEffect(() => {
 		const dbf = debounce(() => runSearch(page), 750);
 		dbf();
 		return () => dbf.cancel();
-	}, [selectedOption, searchText, page]);
+	}, [selectedOption, searchText, page, dateField, orderingForce]);
 
 	const chefsPagination: IPagination | null = useSelector((state: State) =>
 		chefSelectors.selectPagination(state),
 	);
 
 	const hasPages = (chefsPagination?.totalPages ?? 0) > 1;
 
+	const getUpdateConfig = () => {
+		switch (orderingForce) {
+			case 'default':
+				return undefined;
```

Comment:
To be more explicit about how unknown values are handled, could we add a default case here?

```suggestion
			case 'default':
			default:
				return undefined;
```